src/stubs/common/opamCommonStubs.mli

```diff
@@ -1 +0,0 @@
-external check_executable : string -> unit = "opam_check_executable"
```

Comment:
These files were added in previous commit, should we squash this commit with 4e9b31230e7db1973407c35b772f02d947dc98d3 ?